src/components/SideBar/Sidebar.tsx

```diff
@@ -26,6 +26,7 @@ const SideBar = () => {
               classes[`${pathname === link ? 'c-sidebar__options-item--active' : ''}`]
             }`}
             to={link}
+            key={name}
```

Comment:
Why do you use the name as a key, if we have the same names it produces conflicts in the DOM tree.
Replace name value with index from map function. This is not the best solution but in our case, it is good. In general, we need to provide a unique and stable keys for the list's elements.